packages/ui/src/overview/components/DeadlineList/ElectionListItem.tsx

```diff
@@ -48,39 +57,52 @@ export const ElectionListItem: React.FC<ElectionListItemProps> = React.memo(({ e
     }
   }, [])
 
-  const copyMessage = () => {
-    const urlAddress = '#/election?candidate='
+  const getCopyMessage = () => {
```

Comment:
That's my mistake, but it would make more sense to have a `useMemo` here:
```suggestion
  const getCopyMessage = useMemo(() => {
    ...
  }, [electionStage, election.candidates.length, canVote, votes])
```